### PR TITLE
GS-75: Link save and cancel buttons to settings page

### PR DESCRIPTION
**Pull request by gochoa1088**
This PR closes https://spacem8s.atlassian.net/browse/GS-75

### CODE REVIEW

**Review by vincentochoa3 — APPROVED** [review]
Clean

File: components/AvatarList/index.js [hunk 1]
```diff
@@ -18,8 +20,16 @@ const AvatarList = () => {
         </div>
       </div>
       <div className={styles.buttonContainer}>
-        <button>Cancel</button>
-        <button>Save</button>
+        <Link href="/settings">
+          <a>
+            <button>Cancel</button>
```
Comment (joelcolucci) [inline components/AvatarList/index.js:25]: Nice work using the Next.js <Link> component!

We can remove the button element altogether.

Styling can be applied by add a class to the anchor tag.

Comment (joelcolucci) [reply components/AvatarList/index.js:25]: *I know this is merged. Sharing for reference moving forward.

Comment (gochoa1088) [reply components/AvatarList/index.js:25]: Thanks Joel.

I think I might create a LinkedButton component to be reused throughout the application, correctly omitting the button element and styling the anchor tag appropriately.

For corrections like this that are required app-wide, should we make the changes in each individual branch thus creating several new PRs? Or can I make a new ticket/branch addressing the creation of the new reusable component and the corrections needed?

**Pull request merged**